models/Job.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+
+const JobSchema = new mongoose.Schema(
```

Comment:
This works just keep in mind for the final project this'll have to be extended.  I'm sure you're probably already aware of this since you did this work a long time ago!